taipy/gui/_renderers/factory.py

```diff
@@ -153,6 +154,25 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "time": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="TimeSelector",
+            attributes=attrs,
+            default_value=datetime.fromtimestamp(0),
+        )
+        .set_value_and_default(var_type=PropertyType.date)
```

Comment:
Should we have a different type for Time ?

Reply:
Yes we should and done